x-pack/legacy/plugins/uptime/public/state/reducers/ping_list.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { handleActions, Action } from 'redux-actions';
+import { PingsResponse } from '../../../common/runtime_types';
+import { getPings, getPingsSuccess, getPingsFail } from '../actions';
+
+export interface PingListState {
+  pingList: PingsResponse;
+  errors: Error[];
```

Comment:
i think this shouldn't be an array, it should be just an Error type, we only need to keep the last error.

Reply:
Fixed in https://github.com/elastic/kibana/pull/59392/commits/0995904099c996ea47be82b5901969b26ca6f55d

Reply:
Updated in https://github.com/elastic/kibana/pull/59392/commits/0995904099c996ea47be82b5901969b26ca6f55d.